template/apps.yaml.jinja

```diff
@@ -5,10 +5,10 @@ metadata:
   finalizers:
     - resources-finalizer.argocd.argoproj.io
 spec:
-  project: {{ argocd_project }}
+  project: {{ cluster_namespace }}
```

Comment:
For general use - Im not sure its a safe assumption that the name of the project=namespace

Reply:
But it is at DLS and when we come to write docs for ArgoCD we should recommend this way of deploying it.

However I also take Gary's point about half way between specific and general. I did consider making this DLS only for the moment.

Reply:
resolving this and continuing discussion in main thread